### PR TITLE
editor: add endpoint to get workflow and schema

### PR DESCRIPTION
**Pull request by harunurhan**
Ref #1367

### CODE REVIEW

File: inspirehep/modules/editor/views.py [hunk 1]
```diff
@@ -54,3 +55,23 @@ def preview():
...
+@blueprint.route('/holdingpen/<int:workflow_id>', methods=['GET'])
+@login_required
+def get_workflow_and_schema(workflow_id):
+    workflow = workflow_object_class.get(workflow_id)
+    schema_url = workflow.data['$schema']
+    schema_name = (
+        six.moves.urllib.parse.urlsplit(schema_url)
+        .path.split("/")[-1]
+        .split(".")[0]
+    )
```
Comment (michamos) [inline inspirehep/modules/editor/views.py:69]: I don't think you need this, you can pass the URL directly to `load_schema` which will try to extract the schema from there, using https://github.com/inspirehep/inspire-schemas/blob/ab35f92f667cea0974cb54f717ef6913ba30cf65/inspire_schemas/utils.py#L595

Comment (harunurhan) [reply inspirehep/modules/editor/views.py:69]: well that's not that important here that much, but I actually took it from `inspirehep` I think we should replace it there too!

Comment (michamos) [reply inspirehep/modules/editor/views.py:69]: I agree :+1:

Comment (harunurhan) [reply inspirehep/modules/editor/views.py:69]: @michamos perhaps it would be good to make `load_schema` to accept absolute schema urls too

Comment (harunurhan) [reply inspirehep/modules/editor/views.py]: turns out we really need the name in that case and it's not for `load_schema`

File: inspirehep/modules/editor/views.py [hunk 2]
```diff
@@ -54,3 +56,18 @@ def preview():
     endpoint = get_endpoint_from_record(record)
     template = current_app.config['RECORDS_UI_ENDPOINTS'][endpoint]['template']
     return render_template(template, record=record)
+
+
+@blueprint.route('/holdingpen/<int:workflow_id>', methods=['GET'])
+@login_required
```
Comment (drjova) [inline inspirehep/modules/editor/views.py:62]: Maybe just for superusers and curators? That means anyone logged can retrieve info about this workflow

**Review by drjova — APPROVED** [review]
LGTM, just the comment. It's up to you

**Review by drjova — APPROVED** [review]

**Pull request merged**